lib/webmachine/response.rb

```diff
@@ -48,11 +48,9 @@ def set_cookie(name, value, attributes = {})
       cookie = Webmachine::Cookie.new(name, value, attributes).to_s
       case headers['Set-Cookie']
       when nil
-        headers['Set-Cookie'] = cookie
-      when String
-        headers['Set-Cookie'] = [headers['Set-Cookie'], cookie]
+        headers['Set-Cookie'] = [cookie]
       when Array
-        headers['Set-Cookie'] = headers['Set-Cookie'] + cookie
+        headers['Set-Cookie'] = headers['Set-Cookie'] << cookie
```

Comment:
Yea, you're right. I just replaced the operator and didn't really think more about it. Just committed an update.